src/app/InteractionModelEngine.h

```diff
@@ -146,6 +150,7 @@ class InteractionModelEngine : public Messaging::ExchangeDelegate
     CommandSender mCommandSenderObjs[CHIP_MAX_NUM_COMMAND_SENDER];
     ReadClient mReadClients[CHIP_MAX_NUM_READ_CLIENT];
     ReadHandler mReadHandlers[CHIP_MAX_NUM_READ_HANDLER];
+    reporting::ReportingEngine mReportingEngine;
```

Comment:
same for the namespace: reporting::Reporting stutters.

Reply:
agree, updated, thanks